dev.exs

```diff
@@ -132,6 +132,9 @@ Beacon.Pages.create_page!(%{
     <h1 class="text-violet-900">Dev</h1>
     <p class="text-sm">Page</p>
     <%= my_component("sample_component", val: 1) %>
+    <div>
+      <BeaconWeb.Components.image beacon_attrs={@beacon_attrs} name="dockyard_1.png" width="200px" />
```

Comment:
In order to resolve the asset path we need the `router`, see https://github.com/BeaconCMS/beacon/blob/90ec7814e05856be97717c76a81cf653ae3aba49/lib/beacon_web/components/components.ex#L24 and https://github.com/BeaconCMS/beacon/blob/90ec7814e05856be97717c76a81cf653ae3aba49/lib/beacon/router.ex#L159-L162

Another option was to inject beacon's assigns into socket private assigns and pass `@socket` instead, which is already available and has more data that we may need in the future but that seems overkill. Passing `@beacon_attrs` we make it explicit what that struct holds https://github.com/BeaconCMS/beacon/blob/90ec7814e05856be97717c76a81cf653ae3aba49/lib/beacon/beacon_attrs.ex and we can provide better docs too.